stdlib/2/typing.pyi

```diff
@@ -385,7 +385,8 @@ class Pattern(Generic[AnyStr]):
 
 # Functions
 
-def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+def get_type_hints(obj: Callable, globalns: Optional[dict[AnyStr, Any]],
```

Comment:
(The reason is that AnyStr is defined as a type variable; see the [mypy docs](http://mypy.readthedocs.io/en/latest/generics.html#type-variables-with-value-restriction).)